style.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 800px) {
+    body {
+      background-color: lightpink;
+    }
+  }
+  
```

Comment:
Nice job pointing this out :+1: However, It would be better if you could make suggestions instead of giving an order to sound nicer.